src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
My preference is for  "deprecated" because it better looks like a state
change, and also because it's what we'll be grepping for in the code base
when looking for, well, deprecations :-)